test/unit/options.test.js

```diff
@@ -97,4 +97,27 @@ describe('options', function () {
             done();
         });
     });
+
+    describe('Option --ignore-iteration ', function () {
+        it('should be undefined by default', function (done) {
+            options({
+                iterationData: './test/fixtures/run/spaces/data.json'
+            }, function (err, result) {
+                expect(err).to.be.null;
+                expect(result.igonoreiteration).to.be.undefined;
+                done();
+            });
+        });
+
+        it('should work correctly when specified', function (done) {
+            options({
+                iterationData: './test/fixtures/run/spaces/data.json',
+                ignoreiteration: 1
+            }, function (err, result) {
+                expect(err).to.be.null;
+                expect(result.ignoreiteration).to.equal(1);
+                done();
+            });
+        });
+    });
```

Comment:
We should also check cases where ignoreIteration has invalid values like `-1`, `> length of iteration data`, `Infinity`, etc.